source/operations/concepts/erasure-coding.rst

```diff
@@ -102,87 +171,8 @@ The following table lists the outcome of varying erasure code parity levels on a
-      MinIO sets the ``STANDARD`` parity based on the number of volumes in the Erasure Set:
-
-      .. list-table::
-         :header-rows: 1
-         :widths: 30 70
-         :width: 100%
-
-         * - Erasure Set Size
-           - Default Parity (EC:N)
-
-         * - 5 or Fewer 
-           - EC:2
-
-         * - 6 - 7
-           - EC:3
-
-         * - 8 or more 
-           - EC:4
-
-      You can override the default ``STANDARD`` parity using either:
-
-      - The :envvar:`MINIO_STORAGE_CLASS_STANDARD` environment variable, *or*
-      - The :mc:`mc admin config` command to modify the ``storage_class.standard`` configuration setting.
-
-      The maximum value is half of the total drives in the :ref:`Erasure Set <minio-ec-erasure-set>`. 
-      The minimum value is ``2``.
-
-      ``STANDARD`` parity *must* be greater than or equal to ``REDUCED_REDUNDANCY``. 
-      If ``REDUCED_REDUNDANCY`` is unset, ``STANDARD`` parity *must* be greater than 2.
-
-   .. tab-item:: REDUCED_REDUNDANCY
-
-      The ``REDUCED_REDUNDANCY`` storage class allows creating objects with lower parity than ``STANDARD``. 
-      ``REDUCED_REDUNDANCY`` requires *at least* 5 drives in the MinIO deployment. 
-      
-      MinIO sets the ``REDUCED_REDUNDANCY`` parity to ``EC:2`` by default.
-      You can override ``REDUCED_REDUNDANCY`` storage class parity using either:
-
-      - The :envvar:`MINIO_STORAGE_CLASS_RRS` environment variable, *or*
-      - The :mc:`mc admin config` command to modify the ``storage_class.rrs`` configuration setting.
-
-      ``REDUCED_REDUNDANCY`` parity *must* be less than or equal to ``STANDARD``.
-
-MinIO references the ``x-amz-storage-class`` header in request metadata for determining which storage class to assign an object. 
-The specific syntax or method for setting headers depends on your preferred method for interfacing with the MinIO server.
-
-- For the :mc:`mc` command line tool, certain commands include a specific option for setting the storage class. 
-  For example, the :mc:`mc cp` command has the :mc-cmd:`~mc cp storage-class` option for specifying the storage class to assign to the object being copied.
-
-- For MinIO SDKs, the ``S3Client`` object has specific methods for setting request headers. 
-  For example, the ``minio-go`` SDK ``S3Client.PutObject`` method takes a ``PutObjectOptions`` data structure as a parameter.
-  The ``PutObjectOptions`` data structure includes the ``StorageClass`` option for specifying the storage class to assign to the object being   created.
-
-
-.. _minio-ec-bitrot-protection:
-
-Bit Rot Protection
-------------------
-
-.. TODO- ReWrite w/ more detail.
+Bitrot Protection
+-----------------
 
 Silent data corruption or bit rot is a serious problem faced by data drives resulting in data getting corrupted without the user’s knowledge. 
 The corruption of data occurs when the electrical charge on a portion of the drive disperses or changes with no notification to or input from the user.
```

Comment:
(Not sure how to suggest changes to more than one line.)

This wording feels like I'm about to be sold a personal care product for my embarrassing data problem. Here's an attempt at a rewrite:

> Bit rot is silent data corruption from random changes at the storage media level.
> For data drives, it is typically the result of decay of the electrical charge or magnetic orientation that represents the data.
> Bit rot can cause subtle errors or corruption with no obvious cause and without warning.

Maybe even provide a link https://en.wikipedia.org/wiki/Data_degradation

Reply:
I'll take it!